src/twisted/internet/_sslverify.py

```diff
@@ -1440,12 +1515,32 @@ def __init__(self,
         @param acceptableProtocols: The protocols this peer is willing to speak
             after the TLS negotiation has completed, advertised over both ALPN
             and NPN. If this argument is specified, and no overlap can be found
-            with the other peer, the connection will fail to be established.
-            If the remote peer does not offer NPN or ALPN, the connection will
-            be established, but no protocol wil be negotiated. Protocols
-            earlier in the list are preferred over those later in the list.
+            with the other peer, the connection will fail to be established. If
+            the remote peer does not offer NPN or ALPN, the connection will be
+            established, but no protocol wil be negotiated. Protocols earlier
+            in the list are preferred over those later in the list.
         @type acceptableProtocols: L{list} of L{bytes}
 
+        @param atLeastTLSVersion: The minimum TLS version that you want to use,
+            or Twisted's default if it is higher. Use this if you want to make
+            your client/server more secure than Twisted's default, but will
+            accept Twisted's default instead if it moves higher than this
+            value. You probably want to use this over C{noLowerThanTLSVersion}.
+        @type atLeastTLSVersion: L{TLSVersion} constant
+
+        @param noLowerThanTLSVersion: The minimum TLS version to use. If not
+            specified, it is a generally considered safe default (TLSv1.0). If
+            you want to raise your minimum TLS version to above that of this
+            default, use C{atLeastTLSVersion}.
+        @type noLowerThanTLSVersion: L{TLSVersion} constant
+
+        @param reduceSecurityToTLSVersion: The maximum TLS version to use. If
```

Comment:
This might warrant some emphasis or some ALL CAPS or something - don't touch unless you know what you're doing, "reduce security" is not a joke, etc.